src/main/java/frc/robot/stateManagement/RobotStateManager.java

```diff
@@ -52,6 +52,14 @@ public NoteState getNoteState() {
   }
 
   // Placement Mode
+  public void switchPlacementMode() {
+    if (this.placementMode.equals(PlacementMode.SPEAKER)) {
```

Comment:
A few things:
1. I think the `this.` prefix is unnecessary here, and you're not using it when you use `placementMode` below. This looks odd and could make people do a double-take (IOW it made me question whether this code was doing what it appears to be doing). I'd suggest removing the `this.` prefix, or if you prefer you could add it to the other uses of `placementMode` below.
2. Given that `placementMode` is an enumeration, you can compare it using `==`; you don't need `.equals()`
3. Given the above, this might be done more concisely with a ternary operator. E.g.:

`placementMode = placementMode == PlacementMode.SPEAKER ? PlacementMode.AMP : PlacementMode.SPEAKER;`

Or if you prefer to split it across multiple lines, I like this formatting for ternaries:

```
placementMode = placementMode == PlacementMode.SPEAKER
  ? PlacementMode.AMP
  : PlacementMode.SPEAKER;
```